view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -22,6 +22,7 @@ define(
         priceUtils,
     ) {
         'use strict';
+        const minimumInstallmentValue = 3000;
```

Comment:
Not a good name really. It is the minimum value of the purchase for instalment payments to be available, not the minimum instalment value -  an instalment is the monthly payment

Reply:
Maybe `installmentMinimumPurchaseAmount` or something (although that is a bit long)

Reply:
ok.. changed.. it is bit long.. but temporary.. so can be